cmd/expose.go

```diff
@@ -50,22 +53,31 @@ ktunnel expose redis 6379
 		nodeSelectorTags := map[string]string{}
 		for _, tag := range NodeSelectorTags {
 			parsed := strings.Split(tag, "=")
-			log.Error(parsed)
 			if len(parsed) != 2 {
 				log.Errorf("failed to parse node selector tag: %v", tag)
 			} else {
 				nodeSelectorTags[parsed[0]] = parsed[1]
 			}
 		}
 
+		deploymentLabels := map[string]string{}
+		for _, label := range DeploymentLabels {
+			parsed := strings.Split(label, "=")
+			if len(parsed) != 2 {
+				log.Errorf("failed to parse deployment label: %v", label)
+			} else {
```

Comment:
no need for else, just `continue` from inside the if

Reply:
I just replicated what you did with nodeSelector, sure update coming 👍

Reply:
haha makes perfect sense :D